fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
we could do that, but what would that common package be? It seems like creating a new shared artifact would add more complexity than the duplication of the code here. Personally, I don't like this filter at all, but I'm hesitant about removing it -- the better solution is to inject a small route in between the fedora message stream and the camel-toolbox apps, but that implies more complexity for people just trying to get something working.